sycl/CMakeLists.txt

```diff
@@ -27,9 +27,14 @@ include(SYCLUtils)
 
 # The change in SYCL_MAJOR_VERSION must be accompanied with the same update in
 # llvm/clang/lib/Driver/CMakeLists.txt.
+#
+# For the current development cycles MINOR has been bumped up and MAJOR wasn't.
```

Comment:
Suggest removing this line as it can be misleading if anybody forgets to update it.

Reply:
Well, I think we have the information to know what has been changed during the current cycle:

  MINOR == 0   ==> MAJOR has been incremented
  MINOR != 0    ==> It has been increased as part of previous release before this cycle started.

so, technically, we can remove it. I'm not sure if we want though.

@ intel/llvm-reviewers-runtime, what do others think?

Reply:
I've had an offline chat with @steffenlarsen (based on the doc counterpart PR) and he asked essentially the same from a slightly different angle. I'll make the change.